### ISSUE TITLE
index.txt instructions

### ISSUE THREAD

**Issue by RyanZurrin** [pnlbwh/luigi-pnlpipe#70]
Add documentation on how to create a proper index.txt file for synb0 or other tasks.

**Comment by tashrifbillah** [comment]
We had referred users to FSL documentation in the past who are the creators of index.txt:

http://ftp.nmr.mgh.harvard.edu/pub/dist/freesurfer/tutorial_packages/centos6/fsl_507/doc/wiki/eddy(2f)UsersGuide.html#A--index

**Comment by RyanZurrin** [comment]
yes but this requires the users to first try running things and then to come to us to ask, I am thinking if you just include a paragraph that says:

"To set up the index.txt, use fslhd or fslinfo to check dim4 value of your data. The Value of dim4 will determine how many 1's seperated by spaces there are. For example say there is dim4=6, then the index.txt will be:
```1 1 1 1 1 1````
"
